runtimes/php/php-apps.md

```diff
@@ -41,6 +41,8 @@ Set the `CC_PHP_VERSION` environment variable to one of the following values:
 - 7.3
 - 7.4
 
+You can also set the `CC_COMPOSER_VERSION` to `1` or `2` to select the composer version to use.
```

Comment:
There is a `composer` section in this page, maybe it could be better in that section? https://www.clever-cloud.com/doc/php/php-apps/#composer

Reply:
I agree. I'll move these line in the right section. As it's on the new doc, I'll close this issue and back-porting changes to the right page.